octobot_trading/exchanges/implementations/spot_ccxt_exchange.py

```diff
@@ -43,26 +62,26 @@ async def create_order(self, order_type: enums.TraderOrderType, symbol: str, qua
             return self.clean_order(created_order)
 
         except ccxt.InsufficientFunds as e:
-            self._log_error(e, order_type, symbol, quantity, price, stop_price)
+            self.connector.log_error(e, order_type, symbol, quantity, price, stop_price)
             self.logger.warning(str(e))
             raise errors.MissingFunds(e)
         except ccxt.NotSupported:
             raise errors.NotSupported
         except Exception as e:
-            self._log_error(e, order_type, symbol, quantity, price, stop_price)
+            self.connector.log_error(e, order_type, symbol, quantity, price, stop_price)
```

Comment:
why logging from connector ?

Reply:
I wanted to keep the previous behaviour, do you think SpotCCXTExchange should use its own logger ?

Reply:
I think it will make it confusing if a code is logging as if it was from somewhere else

Reply:
Yes :+1: I'll add a new logger